code/index.html

```diff
@@ -4,34 +4,108 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="./normalize.css" />
     <title>Technigo Project Cluedo</title>
   </head>
   <body>
+    <!-- BACKGROUND -->
+    <div class="background">
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+    <!-- SOURCE for Background idea, which was modified from the original: https://dev.to/vincenius/websites-for-creative-backgrounds-1mao -->
+    <div class="title">
+      <h1>CLUEDO</h1>
+    </div>
+    <div class="title-p">
+      <p>
+        Choose your suspect, the room and a weapon to reveal the crime.
+        <br />You may shuffle the cards as many times as you'd like to, but the
```

Comment:
I remember Maks mentioned that it is preferred to use several `<p>` instead of `<br>` in the code. I think it is up to you, but just FYI. 🤗